spring-auto-restdocs-json-doclet-jdk9/src/test/java/capital/scalable/restdocs/jsondoclet/DocumentedClass.java

```diff
@@ -24,6 +24,15 @@
 /**
  * This is a test class. UTF 8 test: 我能吞下玻璃而不伤身体。Árvíztűrő tükörfúrógép
  *
+ * Usage example:
+ * <pre>
+ * {@code
```

Comment:
please add test with some text between pre and @code
also test for some more crazy stuff, like:
`<code><pre>{@code <code>mycode}</pre></code>`